server/util/disk/disk.go

```diff
@@ -15,7 +15,7 @@ import (
 
 func EnsureDirectoryExists(dir string) error {
 	if _, err := os.Stat(dir); os.IsNotExist(err) {
-		return os.MkdirAll(dir, 0755)
+		return os.MkdirAll(dir, 0750)
```

Comment:
I am a bit worried that this might be a breaking change, but Tyler or Siggi might have better intuition here.

Reply:
That's totally fair, I'm not sure what our permissions need to be there. @tylerwilliams do you know?